cumulus_library/studies/core/builder_encounter_coding.py

```diff
@@ -168,3 +117,4 @@ def prepare_queries(self, cursor: object, schema: str):
                         table_cols=["id", "code", "code_system", "display"],
                     )
                 )
+        self.write_queries()
```

Comment:
Is this a debug tool that got left in or was this an important step that was missing before? :smile:

Reply:
debug, will trim.